DigitalExcellence/dex-frontend
# Styling contributor overview

## Description

Fixed the styling of the contributor overview on the project detail page. 

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Go to the project detail page
2. Check if the contributor section is correct according to the design (font, text color, alignment)


## Link to issue

Closes: #182
